adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js

```diff
@@ -263,6 +263,76 @@ var AdyenCheckoutHybris = (function () {
             }
         },
 
+        initiatePaypal: function (amount) {
+            var paypalNode = document.getElementById('adyen_hpp_paypal_container');
+
+            var paypalComponent = this.checkout.create("paypal", {
+                environment: "test", // Change this to "live" when you're ready to accept live PayPal payments
```

Comment:
can we use this from Environment ? like environment.test constant.

Reply:
Yes, already chaged that. Will commit soon.

Reply:
Done